recipe/bld.bat

```diff
@@ -175,46 +172,49 @@ if errorlevel 1 exit /b 1
 
 if "%PKG_NAME%" == "libtorch" (
     @REM Extract the compiled wheel into a temporary directory
-    if not exist "%SRC_DIR%/dist" mkdir %SRC_DIR%/dist
-    pushd %SRC_DIR%/dist
-    for %%f in (../torch-*.whl) do (
+    if not exist "%SRC_DIR%\dist" mkdir %SRC_DIR%\dist
+    pushd %SRC_DIR%\dist
+    for /f %%f in ('dir /b /S ..\torch-*.whl') do (
         wheel unpack %%f
+        if %ERRORLEVEL% neq 0 exit 1
     )
 
-    @REM Navigate into the unpacked wheel
-    pushd torch-*
+    @REM Navigate into the unpacked wheel; naming pattern of the folder is documented:
+    @REM https://github.com/pypa/wheel/blob/0.45.1/src/wheel/cli/unpack.py#L11-L12
+    pushd torch-%PKG_VERSION%
+    if %ERRORLEVEL% neq 0 exit 1
 
     @REM Move the binaries into the packages site-package directory
-    robocopy /NP /NFL /NDL /NJH /E torch\bin %SP_DIR%\torch\bin\
-    robocopy /NP /NFL /NDL /NJH /E torch\lib %SP_DIR%\torch\lib\
-    robocopy /NP /NFL /NDL /NJH /E torch\share %SP_DIR%\torch\share\
+    robocopy /NP /NFL /NDL /NJH /E torch\bin\ %LIBRARY_BIN%\
+    robocopy /NP /NFL /NDL /NJH /E torch\lib\ %LIBRARY_BIN%\ *.dll
+    robocopy /NP /NFL /NDL /NJH /E torch\lib\ %LIBRARY_LIB%\ *.lib
+    robocopy /NP /NFL /NDL /NJH /E torch\share %LIBRARY_PREFIX%\
```

Comment:
I think the syntax here doesn't work as for `cp`, and will need something like
```suggestion
    robocopy /NP /NFL /NDL /NJH /E torch\share\ %LIBRARY_PREFIX%\share
```

Reply:
We should also add a smoketest for a given file that we expect to be under `%LIBRARY_PREFIX%\share`.